### DISCUSSION TITLE
[Script update] Ubuntu VM

### DISCUSSION THREAD

**Discussion by irish1986** [community-scripts/ProxmoxVE#268] [Ideas]
# Application Name
Ubuntu-VM (cloud-image)

## Website
https://cloud-images.ubuntu.com/

## Description
Current there are 2 scripts for [ubuntu2404](https://github.com/community-scripts/ProxmoxVE/blob/main/vm/ubuntu2404-vm.sh) and [ubuntu2204](https://github.com/community-scripts/ProxmoxVE/blob/main/vm/ubuntu2204-vm.sh).  These are very helpful although it would be interesting to create a common script for all Ubuntu release version.

## Action
I will be opening a Pull Request creating a new **ubuntu-vm** script which will ask the end user which version of Ubuntu needs to be installed.  The default will be `noble` and all 4 current supported version of Ubuntu be included.

**Comment by havardthom** [comment]
Per discussions in https://github.com/community-scripts/ProxmoxVE/pull/269 we are keeping two scripts for now
